torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2866,6 +2866,14 @@ at::Tensor AtenXlaType::select(const at::Tensor& self, int64_t dim,
       XLATensor::select(bridge::GetXlaTensor(self), dim, index));
 }
 
+at::Tensor& AtenXlaType::silu_out(const at::Tensor& self, at::Tensor& out) {
```

Comment:
We usually add a cpp test like
https://github.com/pytorch/xla/blob/403919d167efc64e6d86b269fdd1e2c4a69903de/test/cpp/test_aten_xla_tensor.cpp#L3070

which will compare the pytorch result against the xla result. There should be an `torch::silu_out`defined by pytorch already.